SwiftUIProject/SwiftUIProject/PlayerView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  PlayerView.swift
+//  SwiftUIProject
+//
+//  Created by Валентин Коскин on 19.01.2023.
+//
+
+import AVFoundation
+import Foundation
+
+/// Model playerView
+final class PlayerView: ObservableObject {
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let songNames = ["Riders", "Terror"]
+        static let albumImageNames = ["nfs", "nfs2"]
+        static let zeroDoubleValue = 0.0
+        static let zeroValue = 0
+        static let oneValue = 1
+        static let musicType = "mp3"
+    }
+    
+    // MARK: - Public Properties
+    
+    @Published var maxDuration = Constants.zeroDoubleValue
+    @Published var currentDuration = Constants.zeroDoubleValue
+    var songs = Constants.songNames
```

Comment:
```suggestion
    @Published var currentDuration = Constants.zeroDoubleValue
    
    var songs = Constants.songNames
```
отделяй враперы от обычных свойств пустой строкой

Reply:
плюсую